### PR TITLE
Updating lib ui-box@^2.1.1

### PR DESCRIPTION
**Pull request by Pasalietis**
Breaking Changes:
css is no longer an accepted prop for any component (except `<Pane />`) – it won't interpret that prop and will pass it through to the underlying component.

### CODE REVIEW

**Comment by Pasalietis** [comment]
Is this size limit is hard prerequisite or it can be raised?

File: src/tooltip/src/Tooltip.js [hunk 1]
```diff
@@ -199,13 +200,13 @@ export default class Tooltip extends PureComponent {
...
       >
-        {({ css, style, state, getRef }) => (
+        {({ className, style, state, getRef }) => (
           <TooltipStateless
             id={this.state.id}
             appearance={appearance}
             innerRef={ref => getRef(ref)}
             data-state={state}
-            css={css}
+            className={cx(statelessClassName, className)}
```
Comment (mshwery) [inline src/tooltip/src/Tooltip.js]: Can we push this prop change into `TooltipStateless` instead? That way we can maintain the `css` prop api here for this version without breaking changes.

File: src/corner-dialog/src/CornerDialog.js [hunk 2]
```diff
@@ -233,7 +234,7 @@ export default class CornerDialog extends PureComponent {
               backgroundColor="white"
               elevation={4}
               width={width}
-              css={animationStyles}
```
Comment (mshwery) [inline src/corner-dialog/src/CornerDialog.js]: Seems like we could avoid this change if we pushed it into `Pane` and maybe `Text`. Anywhere that applies `css` to a `Box` component _directly_.

File: src/tooltip/src/Tooltip.js (hunk 1 above)
Comment (Pasalietis) [reply src/tooltip/src/Tooltip.js]: I updated this file to be more closely to https://github.com/segmentio/evergreen/pull/549

File: src/corner-dialog/src/CornerDialog.js (hunk 2 above)
Comment (Pasalietis) [reply src/corner-dialog/src/CornerDialog.js]: Doesn't evergreen should follow ui-box way to drop support of css prop ?

Comment (mshwery) [reply src/corner-dialog/src/CornerDialog.js]: Yeah. I think we should mark it as deprecated so devs know to move off of it. Then in the next major version we can remove it completely.

File: src/switch/src/Switch.js [hunk 3]
```diff
@@ -7,12 +8,12 @@ const animationEasing = {
   spring: `cubic-bezier(0.175, 0.885, 0.320, 1.175)`
 }
 
-const handleStyle = {
+const handleStyleClass = {
```
Comment (tusbar) [inline src/switch/src/Switch.js]: Shouldn’t this call `css({ … })`?

File: src/popover/src/Popover.js [hunk 4]
```diff
@@ -372,11 +374,11 @@ export default class Popover extends Component {
               getRef(ref)
             }}
             data-state={state}
-            css={css}
             display={display}
             minWidth={minWidth}
             minHeight={minHeight}
             {...statelessProps}
+            className={cx(statelessProps.className, gcss(css))}
```
Comment (mshwery) [inline src/popover/src/Popover.js]: This might produce `css-nil` class? Do we want `css ? gcss(css) : undefined`?

**Comment by mshwery** [comment]
>Is this size limit is hard prerequisite or it can be raised?

@Pasalietis we can bump it up a tiny bit if needed. Ideally we find other ways to cut it back but with this type of change it makes sense. What's the size limit change?

**Comment by Pasalietis** [comment]
> 
> 
> > Is this size limit is hard prerequisite or it can be raised?
> 
> @Pasalietis we can bump it up a tiny bit if needed. Ideally we find other ways to cut it back but with this type of change it makes sense. What's the size limit change?

```bash
Package size limit has exceeded by 123 B
Package size: 210.12 KB
Size limit:   210 KB
```

File: src/popover/src/Popover.js (hunk 4 above)
Comment (Pasalietis) [reply src/popover/src/Popover.js]: I look at it

File: src/switch/src/Switch.js (hunk 3 above)
Comment (Pasalietis) [reply src/switch/src/Switch.js]: fixed

File: src/popover/src/Popover.js (hunk 4 above)
Comment (Pasalietis) [reply src/popover/src/Popover.js]: fixed

**Comment by mshwery** [comment]
@Pasalietis go ahead and bump the size limit by 1-2 kb :)

**Comment by tusbar** [comment]
Why not update to 3.0.0 while we’re at it?

**Comment by Pasalietis** [comment]
> 
> 
> Why not update to 3.0.0 while we’re at it?

because 3.0.0 have more breaking changing

**Comment by mshwery** [comment]
Yeah I think if we moved to ui-box@^3.0.0 we'll need to major version bump evergreen as well since the classnames will change.

**Comment by mshwery** [comment]
Closes #469

File: docs/src/components/DocsMDXProvider.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { MDXProvider } from '@mdx-js/tag'
+import { MDXProvider } from '../../node_modules/@mdx-js/react'
```
Comment (mshwery) [inline docs/src/components/DocsMDXProvider.js]: This should just be `@mdx-js/react`

**Comment by mshwery** [comment]
so close @Pasalietis !! How can I help you get this over the line?

File: docs/src/components/DocsMDXProvider.js (hunk 5 above)
Comment (Pasalietis) [reply docs/src/components/DocsMDXProvider.js]: fixed

**Comment by Pasalietis** [comment]
> so close @Pasalietis !! How can I help you get this over the line?

i guess it's done now.

**Pull request merged**